app/models/update.server.ts

```diff
@@ -9,17 +13,28 @@ const ENTRY_TYPE_MAP = {
   [attributeEnum.SPELLING_VARIANT]: "spelling_variants",
   [attributeEnum.FIST_NOTE]: "fist_note",
   [attributeEnum.DAGGER]: "dagger",
+  [attributeEnum.MEANING_HEADER]: "null", // type map must be exhaustive
```

Comment:
Why `"null"` though?

Reply:
So unfortunately neither `undefined` or `null` work on their own; we get this error in `updateEntry`:

> A computed property name must be of type 'string', 'number', 'symbol', or 'any'

Hence the quotes. Though perhaps "undefined" is more appropriate. 

Is there a way you'd prefer this to be solved ?

Reply:
We used a `Partial<Record ...>`.